packaging/tags.py

```diff
@@ -81,15 +103,18 @@ def parse_tag(tag):
 
 
 def _normalize_string(string):
+    # type: (str) -> str
     return string.replace(".", "_").replace("-", "_")
 
 
 def _cpython_interpreter(py_version):
+    # type: (Tuple[int, int]) -> str
```

Comment:
Maybe it would be worth doing a type alias: `PythonVersion = Tuple[int, int]` since this specific tuple format is used in multiple places and it would help communicate exactly what's expected to be passed in?